src/OnlineRetailPortal.Web/Models/Category.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineRetailPortal.Web
+{
+    public enum Category
+    {
+        Properties,
+        Cars,
+        Furniture,
+        Mobiles,
+        Bikes,
+        Books,
+        Fashions,
+        Electronics,
+        Others
```

Comment:
weren't we supposed to keep them singular?

Reply:
Category was in future scope and it it already implemented by rishabh

Reply:
done